packages/sdk/src/Rewards.ts

```diff
@@ -37,20 +39,20 @@ export class Rewards {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#list-reward-assignments
 	 */
-	public listAssignments(rewardId: string, params: $FixMe) {
-		return this.client.get(`/rewards/${encode(rewardId)}/assignments`, params)
+	public listAssignments(rewardId: string, params: T.RewardsListAssigmentsParams) {
```

Comment:
`params` parameter should not be required and just fallback to `{}` as per old SDK